zerver/openapi/zulip.yaml

```diff
@@ -7762,6 +7767,13 @@ paths:
               New in Zulip 4.0 (feature level 58).  This capability is
               for backwards-compatibility; it will be required in a
               future server release.
+
+            * `user_settings_object`: Boolean for whether the `update_display_settings`
+              and `update_global_notifications` events are sent to clients.
```

Comment:
This explanation is too terse; we should explain clearly that those events are intended to be replaced by the new `user_settings` event type. Maybe:

"Boolean for whether the client supports the modern `user_settings` event type. If False,
the server will additionally send the legacy `update_display_settings` and `update_global_notifications` 
event types for backwards-compatibility with clients that predate this API migration.

..."

Reply:
Updated the text with this. Thanks for the suggestion.